airbyte-cdk/python/airbyte_cdk/sources/declarative/incremental/datetime_based_cursor.py

```diff
@@ -87,7 +87,7 @@ def __post_init__(self, parameters: Mapping[str, Any]) -> None:
             else datetime.timedelta.max
         )
         self._cursor_granularity = self._parse_timedelta(self.cursor_granularity)
-        self._cursor_field = InterpolatedString.create(self.cursor_field, parameters=parameters)
+        self.cursor_field = InterpolatedString.create(self.cursor_field, parameters=parameters)
```

Comment:
why does `cursor_field` need to be public?

Reply:
https://github.com/airbytehq/airbyte/pull/38099#discussion_r1608749819
I need to access this field and it looks like this field should be exposed to other components